docs/crud/introduction.md

```diff
@@ -1,7 +1,116 @@
 ---
-id: intro
-title: CRUD with Graphback
-sidebar_label: Introduction
+id: overview
+title: CRUD Overview
+sidebar_label: Overview
 ---
 
-## TODO
\ No newline at end of file
+Graphback generates a CRUD API for each model in your GraphQL schema. This gives you queries, mutations and subscriptions that can be instantly be used perform queries and mutations on your data.
+
+Graphback provides an implementation of [GraphQLCRUD](https://graphqlcrud.org) for a standardised CRUD API using common data access patterns.
+
+Custom queries, mutations and subscriptions you create in your schema will be ignored by Graphback during generation, but will still be in your generated schema, giving flexibility to provide your own implementations on the server.
```

Comment:
Vocabulary. 

Schema - result object.
Models - input. Models can contain mutations,queries that need to have a backing implementation. 


```suggestion
Queries, mutations, and subscriptions can be added to your model files and they will be included in the result schema. Graphback will ignore all queries, mutations and subscriptions in model files. They require manual implementation and merging with graphback generated resolvers. 

Graphback is giving the flexibility to provide your own implementations on the server-side, while generating a lot of helpers that will allow you to 

Your custom operations can utilize Graphback generated input types and filters giving developers the ability control their schema as they would build it from scratch, while significantly reducing the amount of boilerplate required to setup it from your data. 
```